docs/docs-content/clusters/public-cloud/azure/required-permissions.md

```diff
@@ -0,0 +1,300 @@
+---
+sidebar_label: "Required Permissions"
+title: "Required Permissions"
+description: "Review the required permissions for deploying clusters on Azure"
+icon: ""
+hide_table_of_contents: false
+tags: ["public cloud", "azure", "permissions"]
+sidebar_position: 100
+---
+
+
+Palette requires a set of permissions to properly deploy and manage the lifecycle of clusters deployed to Azure. We recommend creating a [role assignment](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments) that has the service principal you want to use with Palette and the appropriate [scope level](https://learn.microsoft.com/en-us/azure/role-based-access-control/scope-overview). To ensure that Palette and VerteX can deploy and manage clusters on Azure in all use cases, use a subscription as the scope level for the role assignment.
+
+:::warning
+
+We recommend you don't use the Azure build-in role, [Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor), as the role assigned to the service principal you want Palette to use. Creating a custom role allows you to provide only those permissions required by Palette.
```

Comment:
```suggestion
We recommend against assigning the built-in Azure [Contributor](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#contributor) role to the service principal you want to use in Palette because its permission scope exceeds our requirements. Instead, create a custom role to provide only those permissions that Palette requires.
```